package.json

```diff
@@ -6,7 +6,8 @@
   "license": "MIT",
   "scripts": {
     "sass": "sass scss:css",
-    "build": "rollup -c"
+    "build": "rollup -c",
+    "prod-build": "rollup -c --environment GOOGLE_PLACES_API_KEY=$GOOGLE_PLACES_API_KEY,WEATHER_API_KEY=$WEATHER_API_KEY"
```

Comment:
Does `npm run build` not work?
I thought `rollup -c` would read env variables from `rollup.config.mjs` so you wouldn't really need to add `--environment` flag.

Reply:
if local it's not problem env variables in rollup.config.mjs but in prod, we need to override the env because there is no env file exist. I want to try if this work anyway